crates/testing/src/predicates.rs

```diff
@@ -20,17 +18,42 @@ impl<INPUT> std::fmt::Debug for Predicate<INPUT> {
     }
 }
 
-pub fn exact<TYPES>(event: HotShotEvent<TYPES>) -> Predicate<Arc<HotShotEvent<TYPES>>>
+pub type ConsecutiveEvents<TYPES> = (Arc<HotShotEvent<TYPES>>, Arc<HotShotEvent<TYPES>>);
```

Comment:
So does this only allow two events to occur out of order?  What do you think about just extending this to be a vector of events that we process?

Reply:
It would be nice to extend this to be a vector, but I'm leaning towards just a tuple since it's what's needed for now, and I'd like to merge this soon to avoid more conflicts. 😄